api-reference/_hidden/dxDataGridColumn/calculateGroupValue.md

```diff
@@ -4,16 +4,172 @@ type: String | function(rowData)
 ---
 ---
 ##### shortDescription
-Specifies a field name or a function that returns a field name or a value to be used for grouping column cells.
+A field name by which to group cells or a function that returns a field name or a value to perform this operation. This value is also used to sort **DataGrid**'s groups.
```

Comment:
1. 
>A field name by which to group cells or a function that returns a field name or a value to perform this operation.
I would mix your structure and the initial one: A field name or a function that returns a field's name or value by which to group cells.
2. This value is also used to sort **DataGrid**'s groups - and the name isn't?

Reply:
1) Will apply it.
2) Yes, only value.